doc/architectural/cbmc-architecture.md

```diff
@@ -61,7 +61,7 @@ transformation tools (see \ref other-tools).
 
 For an explanation of the data structures involved in the modeling of a GOTO
 program (the GOTO Intermediate Representation used by CBMC and assorted tools)
-please see \subpage central-data-structures .
+please see [central data structures](central-data-structures.md).
```

Comment:
This change seems to break the link. When I build the documentation site locally it now links to here - 
![image](https://user-images.githubusercontent.com/35029362/223411326-09de0cb1-6441-4d76-821f-7ce40c4ef43a.png)

I think this link should not be updated as it seems to be working already for the generated website and is not included in the list of errors in this issue - https://github.com/diffblue/cbmc/issues/7576

Reply:
Thank you very much for careful checking! It seems that some other warning got me confused. I have now reverted this change.